jujugui/static/gui/src/test/test_ui_state.js

```diff
@@ -846,8 +864,26 @@ describe('UI State object', function() {
           }
         }
       },
+      // Deployment flow urls.
+      '/deploy/': {
+        sectionA: {},
+        sectionB: {},
+        sectionC: {
+          component: 'deploy'
+        },
+      },
+      '/deploy/summary/': {
+        sectionA: {},
+        sectionB: {},
+        sectionC: {
+          component: 'deploy',
+          metadata: {
+            activeComponent: 'summary'
+          }
+        },
+      },
       // Multi section urls.
-      '/inspector/apache2/machine/3/lxc-0': {
+      '/inspector/apache2/machine/3/lxc-0/deploy/foo': {
```

Comment:
We should have a test for the logic around a url with charmbrowser and deploy because they occupy the same space in the gui but could be supplied in a manually written url.